source/includes/entityref/paybill/_clientcorporationrateagreementcardline.md

```diff
@@ -121,56 +119,86 @@ An Entity representing each individual rate on a Client Corporation
         <tr class="even">
             <td>earnCode</td>
             <td>To-one association</td>
-            <td>EarnCode</td>
+            <td>Earn Code associated with the line, based on the EarnCodeGroup.</td>
             <td>X</td>
             <td></td>
         </tr>
         <tr class="odd">
             <td>externalID</td>
             <td>String (100)</td>
-            <td></td>
+            <td>External identifier for the record, used for migrations and back-office Integration.</td>
             <td></td>
             <td></td>
         </tr>
         <tr class="even">
             <td>markupPercent</td>
             <td>BigDecimal</td>
-            <td></td>
+            <td>
+                <p>Used to indicate what the difference is between the payRate and billRate as a percentage.</p>
+                <p>= (billRate - payRate) / payRate</p>
+                <p>(For example, if payRate = 20 and billRate = 28 then there is a 40% markup.)</p>
+                <p>markups get automatically calculated in code based off their respective billRate and payRate.</p>
+            </td>
             <td></td>
             <td></td>
         </tr>
         <tr class="odd">
             <td>markupValue</td>
             <td>BigDecimal</td>
-            <td></td>
+            <td>
+                <p>Used to indicate what the difference is between payRate and billRate displayed as money.</p>
+                <p>= billRate - payRate</p>
+                <p>For example, there's a $10.00 difference between billRate = $20.00 and payRate = $10.00.</p>
+                <p>markups get automatically calculated in code based off their respective billRate and payRate.</p>
+            </td>
             <td></td>
             <td></td>
         </tr>
         <tr class="even">
             <td>migrateGUID</td>
             <td>String (36)</td>
-            <td></td>
+            <td>The Migrate GUID</td>
```

Comment:
Period after GUID. I think we say unused in other pages. Try to match whatever that says.